README.md

```diff
@@ -1,2 +1,48 @@
-# awesome-books
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> My Personal Portfolio
+
+**Additional description about the project and its features.**
```

Comment:
- [ ] On the readme, please update the project name and description to match this project.

Reply:
@mikenath223, i made a mistake and i have fixed the issue, please take a look at it again 
Thank you